client/gutenberg/extensions/map/save.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+class MapSave extends Component {
+	render() {
+		const { className, attributes } = this.props;
+		const { map_style, map_details, points, zoom, map_center, marker_color } = attributes;
+		const pointsList = points.map( point => {
+			const { longitude, latitude } = point.coordinates;
+			const url = 'https://www.google.com/maps/search/?api=1&&query=' + latitude + ',' + longitude;
```

Comment:
`addQueryArgs` from `@wordpress/url` is neat!

https://github.com/WordPress/gutenberg/tree/4fec0d2959199cbcccdfea93f6fbf115583f1cff/packages/url#readme

Reply:
Thanks, this is a good tip. I'm going to defer this until we've merged this since it's non-critical.